src/components/DetailsCard/cardProps.ts

```diff
@@ -0,0 +1,19 @@
+export type SocialMediaNames = 'facebook' | 'twitter' | 'instagram' | 'github';
+
+type SocialMedia = { name: SocialMediaNames; link: string }[];
+
+type Location = { city: string; country: string };
+
+type FullName = { firstName: string; lastName: string };
+
+export type CardProps = {
+  fullName: FullName;
```

Comment:
I think we can pass fullName as a string instead of an object

Reply:
Okay, will change that

Reply:
Changed ✅